cylc/flow/taskdef.py

```diff
@@ -126,15 +126,6 @@ def describe(self):
         """Return title and description of the current task."""
         return self.rtconfig['meta']
 
-    def check_for_explicit_cycling(self):
-        """Check for explicitly somewhere.
-
-        Must be called after all graph sequences added.
-        """
-        if len(self.sequences) == 0 and self.used_in_offset_trigger:
-            raise TaskDefError(
-                "No cycling sequences defined for %s" % self.name)
-
```

Comment:
I've removed this check as I don't think it's helpful. It is common to define unused tasks, especially in large programatic suites. Another use case is to enable dynamic insertion for diagnostic purposes or whatever.